### PR TITLE
HPCC-21790 Archived workunits not appearing

### PR DESCRIPTION
**Pull request by miguelvazq**
Fixes: HPCC-21790

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by GordonSmith — APPROVED** [review]
Code looks fine - I assume the current default is 60 seconds (may want to double check that)?

**Comment by miguelvazq** [comment]
> Code looks fine - I assume the current default is 60 seconds (may want to double check that)?

Looks like so -> https://github.com/hpcc-systems/hpcc-js-comms/blob/master/src/comms/connection.ts#L31

**Comment by miguelvazq** [comment]
@GordonSmith request value was not making it back to the comm layer after another test. Added some logic that took care of that. Please review.

File: esp/src/eclwatch/WUQueryWidget.js [hunk 1]
```diff
@@ -256,6 +256,11 @@ define([
                     retVal.StartDate = date.add(now, "day", retVal.LastNDays * -1).toISOString();
                     retVal.EndDate = now.toISOString();
                 }
+                if (retVal.Type === "archived workunits") {
+                    lang.mixin(retVal, {
+                        timeOutSeconds: 3000
```
Comment (GordonSmith) [inline esp/src/eclwatch/WUQueryWidget.js]: 50 Minutes?

**Review by GordonSmith — COMMENTED** [review]
I suspect 5 minutes was the intention?

**Comment by miguelvazq** [comment]
@GordonSmith good catch. Updated.

**Review by GordonSmith — APPROVED** [review]
Looks good.

**Comment by miguelvazq** [comment]
@richardkchapman ready to merge

**Pull request merged**